front_end/html/navbar_course.html

```diff
@@ -1,21 +1,24 @@
-            <div class="navbar-item has-dropdown is-hoverable">
+            <div class="navbar-item has-dropdown">
                 {% if "course_basics" in globals() and course_basics["exists"] %}
-                    <a class="navbar-link" href="/course/{{ course_basics["id"] }}">
-                      {{ course_basics["title"] }}
-                    </a>
-                {% else %}
-                    {% if len(courses) > 0 %}
+                    <!--<a class="navbar-link" href="/course/ course_basics["id"] ">
```

Comment:
Please remove comments if you don't anticipate that we will use this code in the future.